pages/api/workshop.ts

```diff
@@ -0,0 +1,64 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { table, base } from "../../utils/table"
+import IEvent from '../../utils/IEvent'
+
+const filterStrings = [
+  'Test Event',
+  'The Ultimate Future Event',
+  'Hack Night'
+]
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  const { method } = req;
+  switch (method) {
+    case 'GET':
+      const events:IEvent[] = [];
+      table.select({
+      }).eachPage(function page(records, fetchNextPage) {
+        
+        RecordsLoop:
+        for (const record of records) {
+          const eventName = record.fields['Event Name'] as string
+          const eventDateStr = record.fields['Event Date & Start Time'] as string
+          const eventDate = new Date(eventDateStr)
+          const recapImg = record.fields['Recap Images'] ?? []
+
+          // maybe there is a better way to filter out events
```

Comment:
There is a better way to filter out events: Airtable lets you use `filterByFormula`, where you pass in an [Airtable formula](https://support.airtable.com/docs/formula-field-reference).

In this case, your formula would be something like `OR(FIND("Test Event", {Event Name}), FIND("The Ultimate Future Event", {Event Name}), FIND("Hack Night", {Event Name}))` (I didn't verify the syntax but it's something like that.)

You should also filter out unlisted events. So actually the formula would be `OR({Unlisted} = 0, FIND("Hack Night", {Event Name}))`